src/main/java/cloud/cholewa/App.java

```diff
@@ -1,7 +1,18 @@
 package cloud.cholewa;
 
+import cloud.cholewa.exceptions.GameException;
+import cloud.cholewa.game.logic.GameEngine;
+
+import java.util.InputMismatchException;
+
+import static cloud.cholewa.common.ConsoleControl.showConsoleMessage;
+
 public class App {
     public static void main(String[] args) {
-        System.out.println("Witaj w Kole Fortuny");
+        try {
+            GameEngine.run();
+        } catch (GameException | IllegalArgumentException e) {
+            showConsoleMessage(e.getMessage());
+        }
```

Comment:
Po tym pierwszym cache dorzuciłbym jeszcze drugi - catch (Exception e) { <tu obsluga> } żeby łapał wszystko. Dlaczego tak? Bo w tym momencie w pierwszym cachu lapiemy wyjatki, których się spodziewamy, a w drugim łapiemy wszystko czego się nie spodziewaliśmy :) Na chwilę obecną obsługa będzie taka sama -> system.out.println, ale to z rozwojem programu może się zmienić.

Reply:
OK, dodane